<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fm_Gcolab</class>
 <widget class="QDialog" name="Fm_Gcolab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>444</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_add">
      <attribute name="title">
       <string>Novo Colaborador</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Nome:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Txt_Cnome"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Username:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Txt_Cusername"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Telefone:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Txt_Cphone">
             <property name="placeholderText">
              <string>(##)#####-####</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Senha:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Txt_Cpassword">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>***********</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Tipo de Acesso</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_Cacesso"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="Bt_Cnew">
           <property name="text">
            <string>Novo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Bt_Csave">
           <property name="text">
            <string>Gravar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BT_Ccancel">
           <property name="text">
            <string>Cancelar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_edit">
      <attribute name="title">
       <string>Gestão de Colaboradores</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_19">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Código do usuário:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Txt_GCcod">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Nome:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Txt_GCnome"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Username:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Txt_GCusername"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Senha:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Txt_GCpassword">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="placeholderText">
                <string>******************</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Telefone:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Txt_GCphone"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_17">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Acesso:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Txt_GCacesso"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_18"/>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,1">
           <item>
            <widget class="QPushButton" name="Bt_GCedit">
             <property name="text">
              <string>Gravar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Bt_GCexcluir">
             <property name="text">
              <string>Excluir</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Bt_GCvendas">
             <property name="text">
              <string>Visualizar vendas</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_21">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Filtrar por:</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QRadioButton" name="Rb_GCcod">
                <property name="text">
                 <string>ID colaborador</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="Rb_GCnome">
                <property name="text">
                 <string>Nome</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_20">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Filtrar:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="Bt_GCfiltrar">
                 <property name="styleSheet">
                  <string notr="true">font-size:12pt;</string>
                 </property>
                 <property name="text">
                  <string>Filtrar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLineEdit" name="Txt_GCfiltrar"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="TableView"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
